README.md

```diff
@@ -90,6 +91,34 @@ The following special entries are ignored when calculating the earliest age (the
 - `--populate_header`: names the header of the column to populate with `populate_value`
 - `--populate_value`: the value to populate every entry within the `populate_header` column
 
+## Categorize Parameters
+
+The following parameters can be used to rename CSV-generated output columns and Irida Next fields as follows:
+
+- `--metadata_1_header`: names the scientific host name column header
+- `--metadata_2_header`: names the common host name column header
+- `--metadata_3_header`: names the food product column header
+- `--metadata_4_header`: names the environmnetal source column header
+- `--metadata_5_header`: names the environmental site column header
```

Comment:
This implies the columns are a specific order, which they won't need to be. It's more that `metadata_1_header` names the `metadata_1` column, etc. Overall, five(?) specific columns (and matching headers) are required in the input.